src/components/Composer/index.js

```diff
@@ -307,19 +307,19 @@ class Composer extends React.Component {
      * @param {ClipboardEvent} event
      */
     handlePaste(event) {
-        event.preventDefault();
-
         const isVisible = this.props.checkComposerVisibility();
         const isFocused = this.textInput.isFocused();
 
         if (!(isVisible || isFocused)) {
             return;
         }
 
-        if (this.textInput.id !== event.target.id) {
+        if (this.textInput !== event.target) {
```

Comment:
This check would break the feature itself for main composer

Reply:
Hey! That's the downside of not using Typescript! It works as expected since both are HTML Elements, using id is a wrong implementation since some elements do not have id(like new message) just a quick improvement to ensure right composer was selected. Debugged and validated with multiple active composers

Reply:
For the main composer in the chat report, we won't need it to be focused. cc: @robertKozik Did you miss that too?

Reply:
Any steps to validate? I thinks this is important to ensure paste action is executed in the right composer we don't require focus validation just target

Reply:
This caused a regression https://github.com/Expensify/App/issues/25550. Previously neither the texinput nor the target had an id (both ids were empty strings) and the early return was never hit. After this change (which seems to fix the logic) we return early if the elements are different but this broke a feature.

Reply:
Interesting!! @s77rt so basically there was a logic based on empty ids working? Woah didn't see that coming, btw I like the fix proposed!